.github/workflows/continuous-integration.yml

```diff
@@ -34,6 +35,10 @@ jobs:
           coverage: "pcov"
           ini-values: "zend.assertions=1"
 
+      - name: "Switch to dev dependencies"
+        if: "${{ matrix.php-version == '8.1' }}"
+        run: "composer config minimum-stability dev"
+
```

Comment:
This can be reverted as soon as we release new DBAL versions with the fix for DECIMAL on SQLite.

Reply:
Let's maybe add a link to the relevant DBAL issue and/or PRs then?

Reply:
Done.